Formula/stackql.rb

```diff
@@ -0,0 +1,29 @@
+class Stackql < Formula
+  desc "SQL interface for arbitrary resources with full CRUD support"
+  homepage "https://stackql.io/"
+  url "https://github.com/stackql/stackql.git",
+    tag:      "v0.3.265",
+    revision: "ce58a3b9cd00022da42cb8144922613e9491f2f3"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "1"
+    ldflags = [
+      "-s",
+      "-w",
+      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildMajorVersion=0",
+      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildMinorVersion=3",
+      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildPatchVersion=265",
+      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildCommitSHA=#{Utils.git_head}",
+      "-X github.com/stackql/stackql/internal/stackql/cmd.BuildShortCommitSHA=#{Utils.git_short_head}",
+      "-X \"stackql/internal/stackql/planbuilder.PlanCacheEnabled=true\"",
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags), "--tags", "json1 sqleanall", "./stackql"
+  end
+
+  test do
+    assert_match "stackql v#{version}", shell_output("stackql --version")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
@SMillerDev done  no problems.  Have added one more test that will interrogate the local file system for (remotely hosted and downloadable) interface definitions, and, finding none, return a header line.